src/components/User/Assets/AssetRequests/AssetRequests.Show.vue

```diff
@@ -13,123 +13,123 @@
     </template>
 
     <div
-      class="token-requests-show"
-      v-else-if="tokenRequest.id && !isRejecting"
+      class="asset-requests-show"
+      v-else-if="assetRequest.id && !isRejecting"
     >
       <h2>
-        <template v-if="tokenRequest.type === 'create_asset'">
-          Token creation request
+        <template v-if="assetRequest.type === 'create_asset'">
```

Comment:
Let's look at `js-sdk` and try to find `assetRequest` types as constant. If this information does not exist, please, move these values (`create_asset` and `update_asset`) to some constant.